streams/pe.m3u

```diff
@@ -443,9 +443,13 @@ https://tv.oyotunstream.com:1936/xtv/xtv/playlist.m3u8
 http://177.53.152.186:8000/play/a00g/index.m3u8
 #EXTINF:-1 tvg-id="" status="blocked",K-Pop Channel
 http://177.53.152.186:8000/play/a00j/index.m3u8
-#EXTINF:-1 tvg-id="" status="error",Memorias TV
-https://play.amelbasoluciones.co:3759/live/memoriastvlive.m3u8
 #EXTINF:-1 tvg-id="" status="error",Radio Xtrema (360p) [Not 24/7]
 https://tv.portalexpress.es:3090/stream/play.m3u8
 #EXTINF:-1 tvg-id="" status="error",Wtv (La Verdad y Punto) (Chincha) (720p) [Not 24/7]
 https://v4.tustreaming.cl/wtv/index.m3u8
+#EXTINF:-1 tvg-id="DiasporaTV.pe" status="online",Diáspora TV
+https://s1.tvdatta.com:3804/live/diasporatvlive.m3u8
+#EXTINF:-1 tvg-id="La5ta.pe" status="online",La 5ta [Not 24/7]
+https://srv.panelcast.net/la5ta/la5ta/playlist.m3u8
```

Comment:
Is it normal for the link to broadcast HRTV?
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/7253922/183272569-00acfe0f-8403-4df7-8601-ec62afc85461.png">

Reply:
Previously, la 5ta broadcast its own programming. Now it broadcasts a mirror signal (https://www.facebook.com/La5taOnlineOficial/videos/5677731242271675/), which is rare. It seems to be a duplicate of the Piura broadcast.

Reply:
I would keep it as is. This website has its own Facebook page and website. I'm not sure which one they will keep (La 5ta or HRTV), or if they will merge.